recipe/meta.yaml

```diff
@@ -23,7 +23,6 @@ requirements:
     - python >=3.8
     - click >=7.0
     - rasterio >=1.3.3
-    - numpy >=1.15,<2.dev0
```

Comment:
Rasterio will bring numpy. The pin here was probably due to lack of ,<2 there. (Not sure.) Anyway, the issue is gone and that was not the best way to fix this.

Thanks for correcting it.

Reply:
Thanks so much @ocefpaf!!!

Now I can install rio-cogeo alongside Numpy 2. :tada: